sippy-ng/src/component_readiness/CopyPageURL.js

```diff
@@ -0,0 +1,25 @@
+import { Box, Button, Tooltip, Typography } from '@mui/material'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+// This is used as a convenient way to capture the page URL for use with
+// the triaging workflow.
+export default function CopyPageURL(props) {
+  const handleCopyClick = () => {
+    navigator.clipboard.writeText(props.apiCallStr)
+  }
+
+  return (
+    <Box align="right">
+      <Typography variant="caption">
+        <Tooltip title={props.apiCallStr}>
+          <Button onClick={handleCopyClick}>copy page URL</Button>
```

Comment:
COPY API URL

With that it looks good to me. Page sounds like the current page URL. (i.e. sippy-ng)